Jenkinsfile

```diff
@@ -46,6 +46,8 @@ pipeline {
                     docker exec -u saluser \${container_name} sh -c \"source ~/.setup.sh && cd /home/saluser/repos/ts_config_mttcs && /home/saluser/.checkout_repo.sh \${work_branches}\"
                     docker exec -u saluser \${container_name} sh -c \"source ~/.setup.sh && cd /home/saluser/repos/ts_xml && /home/saluser/.checkout_repo.sh \${work_branches}\"
                     docker exec -u saluser \${container_name} sh -c \"source ~/.setup.sh && cd /home/saluser/repos/ts_idl && /home/saluser/.checkout_repo.sh \${work_branches}\"
+                    docker exec -u saluser \${container_name} sh -c \"source ~/.setup.sh && cd /home/saluser/repos/ts_sal && git reset --hard ab4b50c\"
```

Comment:
This is not a good idea! please, don't tie the build to a particular version of sal (and xml, below). Otherwise we lose the purpose of this build which is catch issues early.

Instead do:

```
source ~/.setup.sh && cd /home/saluser/repos/ts_sal && /home/saluser/.checkout_repo.sh \${work_branches}
```

Reply:
The code you showed is in the Jenkinsfile already. I added the specific commit is because the `ts_xml` `develop` branch is breaking the `ts_salobj` for the `ts_xml` commit: `f6087af`. I documented this in the `version_history.rst` already.

Reply:
Restore the `Jenkinsfile` in commit: b085919. Thanks!